booklist.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  <meta content="utf-8" http-equiv="encoding">
+  <title>Books Cited by Wikipedia</title>
+
+  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="css/common.css">
+  <link rel="stylesheet" type="text/css" href="css/booklist.css">
+</head>
+<body>
+  <table width="100%" style="max-height: 200px; ">
+    <tr class="spaceUnder">
+      <td style="text-align: center;">
+        <div id ="horizontal_line" class="lower_margin"></div>
+        <div class='container'>
+          <img class='resize_fit_center'
+            src='images/logo.gif' />
+        </div>
+        <div id="horizontal_line" style="margin-top:3px"></div>
+      </td>
+    </tr>
+  </table>
+  <div>
+    <h2 style="text-align: center;"><strong>Books Cited by Wikipedia:</strong></h2>
+  </div>
+  <BR>
+  <BR>
+  <div class="wrapper" id="resultsTray">
+  </div>
+</body>
+</html>
+<script src="scripts/booklist.js"></script>
+<script src="scripts/jquery.js"></script>
+<script src="scripts/bootstrap.min.js"></script>
```

Comment:
Why are these scripts outside the doc, shouldn't they be inside html head?

Reply:
I believe I was copying the format from a different html file, but I'll move those now.

Reply:
actually moving them into the head gives a TypeError, which I believe is because the html elements must be rendered before the script gets called

Reply:
No, please leave them as they are now.
I've also checked other files like `annotation.html` and `similarweb.html` and they do the same thing as you with their JS.